composer.json

```diff
@@ -14,7 +14,8 @@
     "require-dev": {
         "phpunit/phpunit": "^9.6",
         "silverstripe/gridfieldqueuedexport": "^4",
-        "symbiote/silverstripe-queuedjobs": "^6"
+        "symbiote/silverstripe-queuedjobs": "^6",
+        "silverstripe/frameworktest": "^2"
```

Comment:
```suggestion
```
We intentionally don't add this to recipes - people can `composer create-project silverstripe/recipe-blog` and we don't want them getting frameworktest in their project dependencies.
That's why it was added to ci.yml in the first place. Add it back there with the correct constraint, and add a comment so we don't try to do this again.

Reply:
Updated